backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/domain/Movie.kt

```diff
@@ -0,0 +1,10 @@
+package com.pk.zbtz.zbtzbackend.domain
+
+data class Movie(
+    val id: Long,
+    val title: String,
+    val directorId: Long,
```

Comment:
It should be a list, there can be more than one director in the film.

Reply:
Changed